apps/artifactory/artifactory-helm/artifactory.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: artifactory-oss
+  namespace: artifactory
+spec:
+  timeout: 20m
+  releaseName: artifactory-oss
+  chart:
+    spec:
+      chart: jfrog/artifactory-oss
+      version: 7.90.9
+      sourceRef:
+        kind: HelmRepository
+        name: artifactory
+        namespace: artifactory
+  interval: 1m
+  values:
+    artifactory:
+      #speicifes the artifactory-oss container image version
+      image:
+        tag: 7.90.9
+      persistence:
```

Comment:
Don't we need to define https://github.com/jfrog/charts/blob/master/stable/artifactory/values.yaml#L507 ? Thought that this is how we get a pvc by using the helm chart

Reply:
https://github.com/jfrog/charts/blob/master/stable/artifactory/templates/artifactory-statefulset.yaml#L1615

This^ which would reflect what we have in flux atm

Reply:
we can also define pic through this var

https://github.com/jfrog/charts/blob/master/stable/artifactory/values.yaml#L720

Reply:
Doing it that way should handle everything we need by [provisioning the PV too ](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#volume-claim-templates)

Reply:
The template ordering is a bit confusing, but [that resource](https://github.com/jfrog/charts/blob/master/stable/artifactory/templates/artifactory-statefulset.yaml#L1595) will get deployed with the stateful set if it's defined in values